addon/components/frost-sort-item.js

```diff
@@ -1,36 +1,58 @@
 import Ember from 'ember'
-import computed from 'ember-computed-decorators'
 import layout from '../templates/components/frost-sort-item'
-import _ from 'lodash/lodash'
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
 
-const {Component, isEmpty} = Ember
+const {
+  A,
+  computed,
+  Component,
+  isEmpty
+} = Ember
 
-export default Component.extend({
+export default Component.extend(PropTypesMixin, {
   layout: layout,
   classNames: ['frost-sort-item'],
 
-  @computed
```

Comment:
So, does using the decorators have a negative impact on coverage? I could see that maybe if we're computing it based on the transpiled code, but once `ember-cli-code-coverage` supports addons, will that still be a concern? 

Agreed that I'd rather use decorators than have higher coverage numbers, especially if it's just an issue with the reporting of the coverage (i.e. You can still test the computed prop).